proxy/envoy/resources.go

```diff
@@ -88,9 +88,9 @@ const (
 	RedisProxyFilter = "redis_proxy"
 
 	// RedisDefaultOpTimeout is the op timeout used for Redis Proxy filter
-	// Currently it is set to 30ms (conversion happens in the filter)
+	// Currently it is set to 30ms
```

Comment:
30s is a more realistic timeout, isn't it?
let's update just the comment to say 30s and not change the timeout value.

Reply:
yes, you are right. please ignore, though. it doesn't have anything to do with the rest of the PR. i'll clean it up